.github/workflows/check-toc.yaml

```
@@ -0,0 +1,21 @@
+name: Check for README updates
+
+on: pull_request
+
+jobs:
+  check-toc:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Verify ToC
+        run: |
+          npx markdown-toc README.md -i
+          if [[ $(git status --porcelain) ]]; then
+              echo "Did you forget to update the README table of contents?" >&2
+              echo "Please run 'npx markdown-toc README.md -i' and update your PR" >&2
```

```suggestion
              echo "Error: Did you forget to update the README table of contents?" >&2
              echo "Error: Please run 'npx markdown-toc README.md -i' and update your PR" >&2
```
I think that in order to make it easy to understand where the error is, we should add `Error` before the message.

---

done